mono/mini/mini-runtime.c

```diff
@@ -1905,6 +1905,97 @@ mono_jit_map_is_enabled (void)
 
 #endif
 
+#if ENABLE_JIT_DUMP
+static FILE* perf_dump_file;
+uint64_t codeIndex = 0;
+pthread_mutex_t mutex = PTHREAD_MUTEX_INITIALIZER;
+void *mmapAddr = MAP_FAILED;
+
+void
+mono_enable_jit_dump (void)
+{
+        if (!perf_dump_file) {
+		char name [64];
+		FileHeader * header = malloc (sizeof (FileHeader));
```

Comment:
This memory is allocated but never freed; this pointer is never passed to any other function. Please allocate this in the stack instead to avoid the leak.